haas/api.py

```diff
@@ -263,6 +263,19 @@ def node_power_off(node):
     node.obm.power_off()
 
 
+@rest_call('POST', '/node/<node>/dev/<dev>/set_bootdev', Schema({
+    'node': basestring, 'dev': basestring,
```

Comment:
*if* we are going to specify a fixed set of supported boot devices as you've documented (which perhaps we shouldn't; see discussion above), then we can tighten up the schema for `dev`; instead of basestring you can just specify `Or('pxe', 'disk', ...)` (after importing Or from schema of course). Regardless, we need to check that the value is legal *somewhere*.